.github/workflows/test.yml

```
@@ -19,9 +19,27 @@ jobs:
 
             - name: Prepare.
               run : |
-                    pip install -r requirements.txt
-                    pip install pytest
+                  pip install -r requirements.txt
+                  pip install pytest
 
             - name: Run tests.
               run : |
-                    PYTHONPATH=$PWD pytest tests/test_topology.py
+                  PYTHONPATH=$PWD/hwloc_xml_parser pytest tests/test_topology.py
```

```suggestion
                  PYTHONPATH=$PWD pytest tests/test_topology.py
```

and write `from hwloc_xml_parser.topology import SystemTopology` in the test ?